tests/containers/podman_ipv6.pm

```diff
@@ -27,6 +27,7 @@ sub _cleanup {
 sub run {
     my ($self, $args) = @_;
 
+    die "This will work only on GCE and EC2. I'm sorry for the inconvenience :(" unless (is_ec2 || is_gce);
```

Comment:
```suggestion
    die "IPv6 is not available on Azure" if (is_azure);
```
Careful, we're mixing different exclusion styles. In `main_containers.pm` we have:

```
loadtest 'containers/podman_ipv6' if (!is_azure && is_sle('>=15-SP5'));
```

And here it is `is_ec2 || is_gce`. I recommend to unify the exclusion settings, or even better not have them in the test run at all. The test run is already excluded, so IMHO this `die` can be removed as a whole.

A comment in the summary is IMHO better suited.

Reply:
Please check now.